### PR TITLE
feat: use graph for revealing past votes

### PR DESCRIPTION
**Pull request by daywiss**
## motivation
We currently arent able to decode historical votes made from the v1 contract.

## changes
This switches the app to defer to the graphs reveal values when looking up past votes, but maintains the onchain decription for active votes. This automatically looks up desinated voting factory address, but we can override if needed with env var.  btw, i realize now theres a serverless function for this, could switch to that if desired. 

image

### CODE REVIEW

**Comment by smb2796** [comment]
Looks good! Approving, but might want to wait for code review from @chrismaree @ryanwolhuter before merging. Also looks like something is breaking in ci

**Review by smb2796 — APPROVED** [review]

**Comment by ryanwolhuter** [comment]
@smb2796 looks like the build is only failing on netlify. it works on vercel and on my local. I actually don't have access to the team's netlify so I can't check what's going on. can you please give me access? also why do we even have this netlify deployment? I don't recall us ever using it?

File: hooks/queries/user/useDesignatedVotingV1Address.ts
Comment (ryanwolhuter) [inline hooks/queries/user/useDesignatedVotingV1Address.ts:9]:
```suggestion
      enabled: !!address && !isWrongChain,
```

we should add an `isWrongChain` check so that this doesn't throw on the wrong chain

**Review by ryanwolhuter — APPROVED** [review]
works perfectly for me. I'll figure out why the build is failing on netlify when I get access. My current guess is something to do with env vars — I know I haven't set any in netlify since afaik we aren't using it

**Comment by daywiss** [comment]
@ryanwolhuter it will fix itself most likely when i push a change

**Pull request merged**
